<launch>
    <node name="lego_manipulation_task_planning" pkg="lego_manipulation" type="lego_manipulation_task_planning" output="screen" required="true">
        <param name="config_fname" value="$(find lego_manipulation)/config/user_config.json"/>
        <param name="root_pwd" value="$(find lego_manipulation)/"/>
    </node>
</launch>